### ISSUE TITLE
Update header spacings so focus rings don't touch

### ISSUE THREAD

**Issue by gfellerph** [swisspost/design-system#7151]
See #6934 for report and discussion. Focus outlines should be considered when spacing elements in the header so a focus ring never touches an active element, otherwise the focus ring color and the active background merge.

### Tasks
- [ ] Update spacings in design
- [ ] Document changes in dev ticket #7152

**Comment by sandra-post** [comment]
- Changed spacings between elements in Global and Local headers from 4px to 8px. 
- Updated Master components in Figma
- Published Figma library with updated components
